Formula/dcm2niix.rb

```diff
@@ -0,0 +1,24 @@
+class Dcm2niix < Formula
+  desc "DICOM to NIfTI converter"
+  homepage "https://www.nitrc.org/plugins/mwiki/index.php/dcm2nii:MainPage"
+  url "https://github.com/rordenlab/dcm2niix/archive/v1.0.20170609.tar.gz"
+  sha256 "17756f28dc42965854b7389d166df6d70c126486887fd2037b99d4a082c037ab"
+  head "https://github.com/rordenlab/dcm2niix.git"
+
+  option "with-dcm2niibatch"
+
+  depends_on "cmake" => :build
+
+  def install
+    if build.with? "dcm2niibatch"
+      system "cmake", ".", "-DBATCH_VERSION=ON", *std_cmake_args
+    else
+      system "cmake", ".", *std_cmake_args
+    end
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/dcm2niix"
```

Comment:
Does this just display the help text? It would be good to have a test that checks more of the functionality.